src/mlpack/core/cv/metrics/top_k_accuracy_impl.hpp

```diff
@@ -0,0 +1,61 @@
+/**
+ * @file top_k_accuracy_impl.hpp
+ * @author Arunav Shandeelya
+ * 
+ * The implementation of class Top_K_Accuracy_score
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_CORE_CV_METRICS_TOPK_ACCURACY_IMPL_HPP
+#define MLPACK_CORE_CV_METRICS_TOPK_ACCURACY_IMPL_HPP
+
+namespace mlpack {
+namespace cv {
+
+template<typename MLAlgorithm, typename DataType, typename TopK>
+double TopKAccuracy::Evaluate(MLAlgorithm& model,
+                        const DataType& data,
+                        const arma::Row<size_t> labels,
+                        const DataType& predsprob,
+                        const TopK& k)
+{
+  if (data.n_cols != labels.n_cols)
+  {
+    std::ostringstream fpp;
+    fpp << "TopK_Accuracy::Evaluate(): number of points (" << data.n_cols << ")"
+        << "doesn't matches number of labels (" << labels.n_cols << ")!"
+        << std::endl;
+    throw std::invalid_argument(fpp.str());
+  }
+  // Taking Classification output from the model.
+  model.Classify(data, predsprob);
+  // Shape of the predicted probabilities.
+  double predictedProb = predsprob.size();
```

Comment:
Is it possible to get the row and column of matrix using `double`?

Reply:
It's the same as using `n_elem`, so yes `double predictedProb = predsprob.size();` works, but in this case `predictedProb[0]` in line 48 should fail.

Reply:
@KimSangYeon-DGU @zoq What alternative can I use for this? Instead, that, can we use the` predsprob.size(1)`,

Reply:
Hey @Shandilya21, The build error occurs because in line 40 and 48, there is an attempt to access a scalar element (double). To fix it, You could the following : 
1. Use armadillo's `.n_cols` and `.n_rows` i.e. `predsprob.n_cols` or `predsprob.n_rows` according to your needs. Keep in mind the armadillo is column major.


You can read more about them in armadillo's [documentation](http://arma.sourceforge.net/docs.html). Let me know if that makes sense to you or if I need to clarify anything. Thanks.
Regards.

Reply:
Thanks. Yeah, I understood and committing changes by looking into the documentation.